specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2020-07-01-preview/extensions.json

```diff
@@ -0,0 +1,736 @@
+          "enum": [
+            "SystemAssigned",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceIdentityType",
+            "modelAsString": false
+          }
+        }
+      },
+      "description": "Identity for the managed cluster."
+    },
+    "ExtensionInstance": {
+      "description": "The Extension Instance object.",
+      "properties": {
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "description": "Properties of an Extension Instance resource",
+          "properties": {
+            "extensionType": {
+              "type": "string",
+              "description": "Type of the Extension, of which this resource is an instance of.  It must be one of the Extension Types registered with Microsoft.KubernetesConfiguration by the Extension publisher."
+            },
+            "autoUpgradeMinorVersion": {
+              "description": "Flag to note if this instance participates in auto upgrade of minor version, or not.",
+              "type": "boolean"
+            },
+            "releaseTrain": {
+              "description": "ReleaseTrain this extension instance participates in for auto-upgrade (e.g. Stable, Preview, etc.) - only if autoUpgradeMinorVersion is 'true'.",
+              "$ref": "#/definitions/ReleaseTrain"
+            },
+            "version": {
+              "description": "Version of the extension for this extension instance, if it is 'pinned' to a specific version. autoUpgradeMinorVersion must be 'false'.",
+              "type": "string"
+            },
+            "scope": {
+              "description": "Scope at which the extension instance is installed.",
+              "$ref": "#/definitions/Scope"
+            },
+            "configurationSettings": {
+              "description": "Configuration settings, as name-value pairs for configuring this instance of the extension.",
+              "type": "object",
+              "$ref": "#/definitions/ConfigurationSettings"
+            },
+            "configurationProtectedSettings": {
+              "description": "Configuration settings that are sensitive, as name-value pairs for configuring this instance of the extension.",
+              "type": "object",
+              "$ref": "#/definitions/ExtensionProtectedSettings"
+            },
+            "installState": {
+              "type": "string",
+              "description": "Status of installation of this instance of the extension.",
+              "$ref": "#/definitions/InstallStateDefinition"
+            },
+            "statuses": {
+              "type": "array",
+              "description": "Status from this instance of the extension.",
+              "items": {
+                "$ref": "#/definitions/ExtensionStatus"
+              }
+            },
+            "creationTime": {
```

Comment:
You can remove creationTime and lastModifiedTime as you are using systemData.

Reply:
Added tasks to remove these properties in new API version
https://msazure.visualstudio.com/One/_workitems/edit/8909577/